.github/workflows/nomad.yaml

```diff
@@ -97,17 +97,23 @@ jobs:
           nix run .#generate-nomad-jobs ${JOB_NAME}
 
       - name: Build scenario
-        run: nix build .#packages.x86_64-linux.${{ matrix.scenario-name }}
+        run: |
+          nix build .#packages.x86_64-linux.${{ matrix.scenario-name }}
 
-      - name: Upload scenario as artifact
+      - name: Upload scenario to bucket
         id: upload-scenario
-        uses: actions/upload-artifact@v6
-        with:
-          path: |
-            ./result/bin/
-            ./result/happs/
-          name: ${{ env.JOB_NAME }}
-          if-no-files-found: error
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.HETZNER_HOLOCHAIN_INFRA_BUCKETS_ACCESS }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.HETZNER_HOLOCHAIN_INFRA_BUCKETS_SECRET }}
+          AWS_DEFAULT_REGION: hel1
+          AWS_ENDPOINT_URL: https://hel1.your-objectstorage.com
+          S3_BUCKET_NAME: wind-tunnel-public-artifacts
+        run: |-
+          OUT_PATH="${S3_BUCKET_NAME}/run-artifacts/${{ github.run_id }}/${{ matrix.scenario-name }}.zip"
```

Comment:
Upload scenario to directory named by github run id. Then when the run completes we can trash the whole directory.